src/boot/ws.js

```diff
@@ -128,7 +128,7 @@ onWebSocket(dbEnum.wsCategory.sessionCreationError, (data) => {
     color: 'negative',
     position: 'top',
     html: true,
-    timeout: 0,
+    timeout: 5000,
```

Comment:
? What are these changes?

Reply:
Is there a bug on the button needing a fix?